boards/DIM/Src/App/App_SevenSegDisplays.c

```diff
@@ -77,5 +81,40 @@ ErrorCode App_SevenSegDisplays_SetHexDigits(
             seven_seg_displays->displays[i], hex_digit);
     }
 
-    return ERROR_CODE_OK;
+    if (seven_seg_displays->set_value_callback != NULL)
+    {
+        seven_seg_displays->set_value_callback();
+    }
```

Comment:
IMO we should just put a check for this in the constructor, I don't see any use case for this where we give it a `NULL` callback.

Reply:
Fine with me :v: